### ISSUE TITLE
Arrows??

### ISSUE THREAD

**Issue by Crisleay** [jameslinimk/chess-com-cheater#4]
I can't see the arrows. Do you have the same error?

**Comment by mu6m** [comment]
did you solve it ?

**Comment by Crisleay** [comment]
no

**Comment by greeboz** [comment]
Yeah unfortunately the arrows are no longer working. Sometimes the cheats don't even tell me the recommended moves on the cheat GUI.;

**Comment by mu6m** [comment]
> Yeah unfortunately the arrows are no longer working. Sometimes the cheats don't even tell me the recommended moves on the cheat GUI.;

it might be a simple styling problem cause i heard the site has been updated

**Comment by Crisleay** [comment]
probably so

**Comment by francogrex** [comment]
I confirm that no arrows are visible. It has not to do with the chess.com site but an problem with this project. If you look carefully, the releases are only partially built, they are missing the arrows functionalities (and other)

**Comment by majid3612** [comment]
> I confirm that no arrows are visible. It has not to do with the chess.com site but an problem with this project. If you look carefully, the releases are only partially built, they are missing the arrows functionalities (and other)

You may use this extension:
https://github.com/majid3612/Chess-Analyzer
